contrib/database/sql/conn.go

```diff
@@ -169,9 +182,11 @@ func (tp *traceParams) tryTrace(ctx context.Context, resource string, query stri
 		opts = append(opts, tracer.Tag(ext.EventSampleRate, tp.cfg.analyticsRate))
 	}
 	span, _ := tracer.StartSpanFromContext(ctx, name, opts...)
+	resource := string(qtype)
 	if query != "" {
 		resource = query
 	}
+	span.SetTag("query_type", string(qtype))
```

Comment:
Thanks Sam! That's not a good move though because that package is meant for constants representing tags which exist cross-language and have meaning in the product. This is more like a helper tag for a specific scenario so it doesn't really feel like it belongs there.

Reply:
Ah, ok. I get you! I can just pop a constant in the database/sql package.

Reply:
Actually ignore that, we will only be using it once (better the tests have it specified manually since they'll need to break if the tag is changed).